### ISSUE TITLE
Problem with dbc2val support for multiple VSS signals in mapping

### ISSUE THREAD

**Issue by erikbosch** [eclipse/kuksa.val.feeders#18]
In [mapping.yml](https://github.com/eclipse/kuksa.val.feeders/blob/main/dbc2val/mapping.yml) we have this

```
DI_uiSpeed:
  vss: # taken from https://github.com/COVESA/vehicle_signal_specification/blob/master/spec/Vehicle/Vehicle.vspec
    datatype: float
    type: sensor
    unit: km/h
    description: Vehicle speed
  databroker:
    datatype: 10 # FLOAT number taken from types_pb2.py
    changetype: 1 # CONTINUOUS number taken from types_pb2.py
  minupdatedelay: 100
  targets: 
    Vehicle.Speed: {}
    Vehicle.OBD.Speed: {}
```

i.e. two VSS signals defined, but databroker setup does not like this and give errors like:

```
2022-08-19 16:47:31,876 WARNING dbcfeeder: Singal DI_uiSpeed has multiple targets: ['Vehicle.Speed', 'Vehicle.OBD.Speed']
2022-08-19 16:47:31,878 INFO databroker: Vehicle.Speed was already registered with id 800, type 10
```

... which could imply that the signal `Vehicle.OBD.Speed` is not regsitered. And later it complains when trying to write `Vehicle.OBD.Speed`

```
022-08-19 16:47:32,080 ERROR dbcfeeder: Exception caught in main loop
Traceback (most recent call last):
  File "/home/erik/kuksa.val.feeders/dbc2val/./dbcfeeder.py", line 258, in _run
    self._provider.update_datapoint(target, value)
  File "/home/erik/kuksa.val.feeders/dbc2val/databroker.py", line 133, in update_datapoint
    id = self._name_to_id[name]
KeyError: 'Vehicle.OBD.Speed'
```

**Comment by SebastianSchildt** [comment]
Yeah, I remember some discussion about this, when we were extending CAN feeder with @DanielGrauel 
The databroker side does not support setting one mapping to multiple locations (which also can be considered a bit of a corner case). I suggest revisiting this, once we are through with https://github.com/eclipse/kuksa.val/issues/308

Then hopefully a lot of code will collapse back into the python sdk, simplifying dbcfeeder a lot

**Comment by SebastianSchildt** [comment]
Duplicate to #28

**Comment by SebastianSchildt** [comment]
Fixed with #33
